CONTRIBUTING.md

```diff
@@ -60,6 +60,12 @@ working on your contributions.
 CoreOS project docs should follow the [Documentation style and formatting
 guide](https://github.com/coreos/docs/tree/master/STYLE.md). Thank you for documenting!
 
+*Note*: rkt embeds footers which report data to Google Analytics. We encourage
```

Comment:
"Some rkt documentation contains a footer that collects data for Google Analytics. The script at `scripts/docs/analytics-footer.py` accepts a `remove` subcommand to delete these footers. It's a good idea to exclude the `Documentation` directory from your git commits to avoid merge conflicts afterwards."

That said, this sounds painful.